model/pdata/common.go

```diff
@@ -820,24 +830,26 @@ func AttributeMapToMap(attrMap AttributeMap) map[string]interface{} {
 	return rawMap
 }
 
+// AttributeMapToMap converts an OTLP AttributeMap to a standard go map
+// Deprecated: use AttributeMap's AsRaw method instead.
+func AttributeMapToMap(attrMap AttributeMap) map[string]interface{} {
+	return attrMap.AsRaw()
+}
+
 // attributeArrayToSlice creates a slice out of a AnyValueArray.
-func attributeArrayToSlice(attrArray AnyValueArray) []interface{} {
-	rawSlice := make([]interface{}, 0, attrArray.Len())
+func attributeArrayToSlice(attrArray AnyValueArray) []AttributeValue {
```

Comment:
Is this change necessary?  It seems like it'll result in `AttributeMap.AsRaw()` converting maps to `map[string]interface{}` while arrays will become `[]AttributeValue`.  Should they both be either `AttributeValue` or `interface{}`?

Reply:
Correct this is no longer necessary with the change to return a `[]interface` in `AsRaw`. Reverted the change.